recipe-server/normandy/settings.py

```diff
@@ -280,7 +280,7 @@ def RAVEN_CONFIG(self):
     SECURE_PROXY_SSL_HEADER = values.TupleValue()
     SECURE_HSTS_SECONDS = values.IntegerValue(3600)
     SECURE_HSTS_INCLUDE_SUBDOMAINS = values.BooleanValue(True)
-    CSRF_COOKIE_HTTPONLY = values.BooleanValue(True)
+    CSRF_COOKIE_HTTPONLY = values.BooleanValue(False)
```

Comment:
To ignore system checks we use `SILENCED_SYSTEM_CHECKS`. You can see an example [here](https://github.com/rehandalal/normandy/blob/42a3404db0fa3e9e635245309b15b246f20a53df/recipe-server/normandy/settings.py#L379).